src/plays/dice-simulator/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Dice Simulator
+
+A dice simulator for RPGs like DnD, in which you can select a range, if you have advantage or not and other features
+
+## Play Demographic
+
+- Language: JavaScript
+- Level: Beginner
+
+## Creator Information
+
+- User: roassaf
+- Gihub Link: https://github.com/roassaf
+- Blog:
+- Video:
+
+## Implementation Details
+
+A plataform where you can simulate dice so that you can play RPGs or other games
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
+
+Update external resources(if any)
```

Comment:
remove these lines if you don't want to use

Reply:
Updated in the new commit